src/ssl/ServerBump.cc

```diff
@@ -44,9 +44,9 @@ Ssl::ServerBump::ServerBump(HttpRequest *fakeRequest, StoreEntry *e, Ssl::BumpMo
 
 Ssl::ServerBump::~ServerBump()
 {
-    debugs(33, 4, HERE << "destroying");
+    debugs(33, 4, "destroying");
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required. Other changes are already discussed in other comments so I will not repeat those arguments here.